docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -275,7 +275,11 @@ MongoClient mongoClient1;
 @Inject
 @MongoClientName("inventory")
 ReactiveMongoClient mongoClient2;
---
+----
+
+[WARNING]
+====
+The old `io.quarkus.mongodb.runtime.MongoClientName` has been deprecated. It still works, but will soon be removed, therefore users are encouraged to not use it.
```

Comment:
You're missing the end of the WARNING.

Reply:
Fixed and changed to `NOTE` as that is what is used in an already existing deprecation message in the doc